xml/System.Collections.Generic/SortedSet`1+Enumerator.xml

```diff
@@ -234,13 +231,13 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- After an enumerator is created or after a <xref:System.Collections.IEnumerator.Reset%2A> is called, <xref:System.Collections.IEnumerator.MoveNext%2A> must be called to advance the enumerator to the first element of the collection before reading the value of <xref:System.Collections.IEnumerator.Current%2A>; otherwise, <xref:System.Collections.IEnumerator.Current%2A> is undefined.  
+ After an enumerator is created or after the <xref:System.Collections.Generic.SortedSet%601.Enumerator.System%23Collections%23IEnumerator%23Reset%2A> method is called, the <xref:System.Collections.Generic.SortedSet%601.Enumerator.MoveNext%2A> method must be called to advance the enumerator to the first element of the collection before reading the value of the <xref:System.Collections.Generic.SortedSet%601.Enumerator.System%23Collections%23IEnumerator%23Current%2A> property; otherwise, <xref:System.Collections.Generic.SortedSet%601.Enumerator.System%23Collections%23IEnumerator%23Current%2A> throws an <xref:System.InvalidOperationException>.  
```

Comment:
This remarks may not be entirely clear since the exception only occurs for `IEnumerator.Current` but not the generic `Current`. Maybe add a comment as such? I know you have the IEnumerator.Current linked here but I wonder if that is adequate.

Reply:
@ianhays do you think readers might 'extrapolate' behavior of `IEnumerator.Current` onto the generic `Current` and vice versa? If that's possible I think the note that highlights differences would be useful. I will add such a note to the both pages: `IEnumerator.Current` and the generic `Current` then. What do you think?

Reply:
My thinking is that yes, readers would extrapolate. Many examples written after generics were introduced show delegating the non-generic interface to the generic implementation:

```csharp
public class C : IEnumerable<string>, IEnumerable
{
    public IEnumerator<string> GetEnumerator<string>() { ... }

    IEnumerator IEnumerable.GetEnumerator() { return this.GetEnumerator<string>(); }
}
```

Similarly with `IEnumerator` and `IEnumerator<T>`.